spokestack/asr/spokestack/keyword_recognizer.py

```diff
@@ -0,0 +1,195 @@
+"""
```

Comment:
it's a bit weird to call this package `keyword_recognizer` and call the similar wakeword package `tflite`
if it's really important for the client to know that they are using tflite, then this module should probably be tflite.py under spokestack/asr/keyword/
if the client doesn't care about tflite, then something like spokestack/asr/keyword/recognizer.py might be simpler
personally, I doubt we will ever implement alternatives to the wakeword/keyword detectors we have now

Reply:
Yeah, I'm not sure if `tflite` is important to the client. Initially, the thought was that we could add support for Onnx/PyTorch. However, in that case, it would be better to not change the top-level interface and just allow a backend swap.